wren-main/src/main/java/io/wren/main/connector/snowflake/SnowflakeSqlConverter.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.wren.main.connector.snowflake;
+
+import com.google.inject.Inject;
+import io.airlift.log.Logger;
+import io.trino.sql.tree.Node;
+import io.wren.base.SessionContext;
+import io.wren.base.sql.SqlConverter;
+import io.wren.main.sqlglot.SQLGlot;
+import io.wren.main.sqlglot.SQLGlotConverter;
+import org.intellij.lang.annotations.Language;
+
+import static io.trino.sql.SqlFormatter.Dialect.SNOWFLAKE;
+import static io.trino.sql.SqlFormatter.formatSql;
+import static io.wren.base.sqlrewrite.Utils.parseSql;
+
+public class SnowflakeSqlConverter
+        implements SqlConverter
+{
+    private static final Logger LOG = Logger.get(SnowflakeSqlConverter.class);
+
+    private final SQLGlotConverter sqlGlotConverter;
+
+    @Inject
+    public SnowflakeSqlConverter(SQLGlot sqlGlot)
+    {
+        sqlGlotConverter = SQLGlotConverter.builder()
+                .setSQLGlot(sqlGlot)
+                .setWriteDialect(SQLGlot.Dialect.SNOWFLAKE)
+                .build();
+    }
+
+    @Override
+    public String convert(@Language("sql") String sql, SessionContext sessionContext)
+    {
+        Node rewrittenNode = parseSql(sql);
+        LOG.info("SnowflakeSqlConverter [Input sql]: %s", sql);
+        String dialectSql = sqlGlotConverter.convert(formatSql(rewrittenNode, SNOWFLAKE), sessionContext);
+        LOG.info("SnowflakeSqlConverter [Dialect sql]: %s", dialectSql);
```

Comment:
```suggestion
        String dialectSql = sqlGlotConverter.convert(formatSql(rewrittenNode, SNOWFLAKE), sessionContext);
```
We have printed the log in `SqlGlotConverter`. I thinks that we don't need to print the similar one again.

Reply:
We format the sql with SNOWFLAKE dialect via `SqlFormatter`. Or we should split the two methods covert and format?

Reply:
hmmm.. oh I found a problem. We shouldn't format SQL by `SNOWFLAKE` dialect here. We should always provide `trino` SQL to `SqlGlot` to get `Snowflake` SQL. We should use `DEFAULT` here.
```suggestion
        String dialectSql = sqlGlotConverter.convert(formatSql(rewrittenNode, DEFAULT), sessionContext);
```
Then, the formatted SQL will printed by `SqlGlotConverter`. We don't need to print it twice.